lib/resize.js

```
@@ -1,19 +1,23 @@
-const Jimp = require('jimp');
-const TinyColor = require('tinycolor2');
+const sharp = require('sharp');
 
 module.exports = async ({ background, type = 'resize', width, height }, buffer) => {
+  type = type === 'resize' ? 'cover' : type;
   // if only one of width/height was provided, make it a square or let Jimp decide if it is able
-  if (width && !height) {
-    height = type === 'resize' ? Jimp.AUTO : width;
+  const resizeOptions = { fit: type };
+  if (width) {
+    resizeOptions.width = width;
   }
-  if (height && !width) {
-    width = type === 'resize' ? Jimp.AUTO : height;
+  if (height) {
+    resizeOptions.height = height;
   }
-  const jimpImage = await Jimp.read(buffer);
-  jimpImage[type](width, height);
   if (background) {
-    const color = new TinyColor(background);
-    jimpImage.background(parseInt(color.toHex8(), 16));
+    resizeOptions.background = background;
+  }
+  try {
+    return sharp(buffer)
+      .resize(resizeOptions)
+      .toBuffer();
+  } catch (e) {
+    console.log(e);
```

What goes on after this? Are we just swallowing the error?